build/patches/experimental-user-scripts-support.patch

```diff
@@ -0,0 +1,10299 @@
+ create mode 100755 components/user_scripts/common/view_type.cc
+ create mode 100755 components/user_scripts/common/view_type.h
+ create mode 100755 components/user_scripts/renderer/BUILD.gn
+ create mode 100755 components/user_scripts/renderer/extension_frame_helper.cc
+ create mode 100755 components/user_scripts/renderer/extension_frame_helper.h
+ create mode 100755 components/user_scripts/renderer/injection_host.cc
+ create mode 100755 components/user_scripts/renderer/injection_host.h
+ create mode 100755 components/user_scripts/renderer/resources/greasemonkey_api.js
+ create mode 100755 components/user_scripts/renderer/resources/user_scripts_renderer_resources.grd
+ create mode 100755 components/user_scripts/renderer/script_context.cc
+ create mode 100755 components/user_scripts/renderer/script_context.h
+ create mode 100755 components/user_scripts/renderer/script_injection.cc
+ create mode 100755 components/user_scripts/renderer/script_injection.h
+ create mode 100755 components/user_scripts/renderer/script_injection_callback.cc
+ create mode 100755 components/user_scripts/renderer/script_injection_callback.h
+ create mode 100755 components/user_scripts/renderer/script_injection_manager.cc
+ create mode 100755 components/user_scripts/renderer/script_injection_manager.h
+ create mode 100755 components/user_scripts/renderer/script_injector.h
+ create mode 100755 components/user_scripts/renderer/scripts_run_info.cc
+ create mode 100755 components/user_scripts/renderer/scripts_run_info.h
+ create mode 100755 components/user_scripts/renderer/user_script_injector.cc
+ create mode 100755 components/user_scripts/renderer/user_script_injector.h
+ create mode 100755 components/user_scripts/renderer/user_script_set.cc
+ create mode 100755 components/user_scripts/renderer/user_script_set.h
+ create mode 100755 components/user_scripts/renderer/user_script_set_manager.cc
+ create mode 100755 components/user_scripts/renderer/user_script_set_manager.h
+ create mode 100755 components/user_scripts/renderer/user_scripts_dispatcher.cc
+ create mode 100755 components/user_scripts/renderer/user_scripts_dispatcher.h
+ create mode 100755 components/user_scripts/renderer/user_scripts_renderer_client.cc
+ create mode 100755 components/user_scripts/renderer/user_scripts_renderer_client.h
+ create mode 100755 components/user_scripts/renderer/web_ui_injection_host.cc
+ create mode 100755 components/user_scripts/renderer/web_ui_injection_host.h
+ create mode 100755 components/user_scripts/strings/userscripts_strings.grdp
+
+diff --git a/chrome/android/BUILD.gn b/chrome/android/BUILD.gn
+--- a/chrome/android/BUILD.gn
++++ b/chrome/android/BUILD.gn
+@@ -235,6 +235,9 @@ android_resources("chrome_app_java_resources") {
+     "//third_party/android_deps:androidx_recyclerview_recyclerview_java",
+     "//third_party/android_deps:material_design_java",
+   ]
++
++  # insert into chrome_app_java_resources
```

Comment:
This comment is unnecessary? Unless you can elaborate on what is `chrome_app_java_resources`?

Reply:
it's just to remind me that that line is relative to

`android_resources ("chrome_app_java_resources") {`

because every now and then wiggle shifts positions and so I remember the right spot.
chrome_app_java_resources are the resources which are then packed in *_percent.pak and placed in the executable folder

Reply:
Ok; please make it a bit more descriptive then?
```suggestion
+  # insert into chrome_app_java_resources which is packed in the *_percent.pak and palced in the executable folder
```

Reply:
fixed in https://github.com/bromite/bromite/blob/8c164138391082350f20493a0ca5c4926887372c/build/patches/experimental-user-scripts-support.patch#L216-L218